client/scss/grids/_grid_base.scss

```diff
@@ -13,6 +13,72 @@
   }
 }
 
+.grid--scroll {
+  position: relative;
+  padding-top: $vertical-space-unit;
+  padding-bottom: 4 * $vertical-space-unit;
+  padding-left: map-get($container-padding, 'small');
+  padding-right: map-get($container-padding, 'small');
+
+  @include respond-to('medium') {
+    padding-bottom: 6 * $vertical-space-unit;
+    padding-left: map-get($container-padding, 'medium');
+    padding-right: map-get($container-padding, 'medium');
+  }
+
+  @include respond-to('large') {
+    padding-left: map-get($container-padding, 'large');
+    padding-right: map-get($container-padding, 'large');
+  }
+
+  @include respond-between('small', 'medium') {
+    // This leaves a comfortable amount of the 'next' grid
+    // item displaying in this configuration
+    // TODO: calculate this based on number of contained grid__cells
+    width: 250%;
```

Comment:
I think the trick involves always keeping c 15% of the second item in the scrollable thing on the screen – I had a look at how you were doing it with galleries, but I didn't think that it looked like the way to go on that count.